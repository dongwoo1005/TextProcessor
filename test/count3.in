stdin count '
There is no single quote
