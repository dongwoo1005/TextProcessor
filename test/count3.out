There
is
no
single
quote
